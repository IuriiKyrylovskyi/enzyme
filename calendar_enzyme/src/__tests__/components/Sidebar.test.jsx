import React from "react";
import { shallow } from "enzyme";
import Sidebar from "../../components/sidebar/Sidebar";

describe("<Sidebar />", () => {
  it("should display component", () => {
    const wrappedComponent = shallow(<Sidebar />);

    expect(wrappedComponent).toMatchSnapshot();
  });
});
