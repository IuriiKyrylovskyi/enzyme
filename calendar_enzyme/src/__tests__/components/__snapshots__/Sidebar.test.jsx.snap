// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> should display component 1`] = `
<div
  className="calendar__time-scale"
>
  <div
    className="time-slot"
    key="9.358485336576194"
  >
    <span
      className="time-slot__time"
    >
      0:00
    </span>
  </div>
  <div
    className="time-slot"
    key="48.987849746002986"
  >
    <span
      className="time-slot__time"
    >
      1:00
    </span>
  </div>
  <div
    className="time-slot"
    key="79.18587476796404"
  >
    <span
      className="time-slot__time"
    >
      2:00
    </span>
  </div>
  <div
    className="time-slot"
    key="7.539399181168371"
  >
    <span
      className="time-slot__time"
    >
      3:00
    </span>
  </div>
  <div
    className="time-slot"
    key="65.38307064244722"
  >
    <span
      className="time-slot__time"
    >
      4:00
    </span>
  </div>
  <div
    className="time-slot"
    key="40.51252396791665"
  >
    <span
      className="time-slot__time"
    >
      5:00
    </span>
  </div>
  <div
    className="time-slot"
    key="83.78623829512222"
  >
    <span
      className="time-slot__time"
    >
      6:00
    </span>
  </div>
  <div
    className="time-slot"
    key="58.719613012052505"
  >
    <span
      className="time-slot__time"
    >
      7:00
    </span>
  </div>
  <div
    className="time-slot"
    key="61.37785748813791"
  >
    <span
      className="time-slot__time"
    >
      8:00
    </span>
  </div>
  <div
    className="time-slot"
    key="34.59701928721783"
  >
    <span
      className="time-slot__time"
    >
      9:00
    </span>
  </div>
  <div
    className="time-slot"
    key="86.66602486846946"
  >
    <span
      className="time-slot__time"
    >
      10:00
    </span>
  </div>
  <div
    className="time-slot"
    key="95.62830261461875"
  >
    <span
      className="time-slot__time"
    >
      11:00
    </span>
  </div>
  <div
    className="time-slot"
    key="7.433452926493134"
  >
    <span
      className="time-slot__time"
    >
      12:00
    </span>
  </div>
  <div
    className="time-slot"
    key="67.47930401292015"
  >
    <span
      className="time-slot__time"
    >
      13:00
    </span>
  </div>
  <div
    className="time-slot"
    key="71.67929451812469"
  >
    <span
      className="time-slot__time"
    >
      14:00
    </span>
  </div>
  <div
    className="time-slot"
    key="44.627087283562325"
  >
    <span
      className="time-slot__time"
    >
      15:00
    </span>
  </div>
  <div
    className="time-slot"
    key="59.123794087336165"
  >
    <span
      className="time-slot__time"
    >
      16:00
    </span>
  </div>
  <div
    className="time-slot"
    key="76.18100821857657"
  >
    <span
      className="time-slot__time"
    >
      17:00
    </span>
  </div>
  <div
    className="time-slot"
    key="24.282221983423913"
  >
    <span
      className="time-slot__time"
    >
      18:00
    </span>
  </div>
  <div
    className="time-slot"
    key="0.8806705207902388"
  >
    <span
      className="time-slot__time"
    >
      19:00
    </span>
  </div>
  <div
    className="time-slot"
    key="98.47821893382516"
  >
    <span
      className="time-slot__time"
    >
      20:00
    </span>
  </div>
  <div
    className="time-slot"
    key="59.7328829901909"
  >
    <span
      className="time-slot__time"
    >
      21:00
    </span>
  </div>
  <div
    className="time-slot"
    key="65.96213337850723"
  >
    <span
      className="time-slot__time"
    >
      22:00
    </span>
  </div>
  <div
    className="time-slot"
    key="53.610226188104136"
  >
    <span
      className="time-slot__time"
    >
      23:00
    </span>
  </div>
</div>
`;
